public/js/script.js

```diff
@@ -41,4 +70,5 @@ function loadEnd(event) {
   console.log(fastQCFile);
   linePlot.render(fastQCFile.modules.qual.quintiles);
   boxPlot.render(fastQCFile.modules.qual.quintiles);
+  parseBasicInfo(fastQCFile.modules.basic)
 }
```

Comment:
I'm not a huge fan of this function performing two tasks; processing the basic-stats data into HTML **as-well-as** inserting it into the DOM. I'm refactoring the code as @starcalibre mentioned in a comment for line 73 to make it more "jquery"-ish, as well as generalising it slightly. 

I'll rename the function to something like; parseAndInsertAsTable.